### PR TITLE
Feat: Quiz Landing page created

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Quiz : home page create
Screenshot_2022-02-21_12-59-15
d

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
Looks Good to me

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+.footer p span {
+    color: crimson;
+}
+
+.footer a {
+    display: flex;
+    background: var(--white);
+    margin: 0 15px;
+    height: 50px;
+    width: 50px;
```
Comment (Megha-Joshi) [inline css/style.css:142]: use rem

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean . Everything looks good to me. You can merge it

**Review by Megha-Joshi — APPROVED** [review]
Good to merge.

**Pull request merged**